AMW_angular/io/src/app/resources/resource-type-add.component.html

```diff
@@ -0,0 +1,30 @@
+<app-modal-header [title]="'Create new Resource Type'" (cancel)="cancel()"></app-modal-header>
+<div class="modal-body">
+  <div class="form-horizontal mx-2">
+    <div class="mb-3">
+      <label for="name" class="form-label">Resource Type name</label>
+      <input type="text" class="form-control" id="name" [(ngModel)]="resourceType.name" />
+    </div>
+    <div class="mb-3">
+      <label for="name" class="form-label">Parent Resource Type (optional)</label>
+      <div ngbDropdown>
```

Comment:
Improve styling of carot 
![2024-12-06_17-33](https://github.com/user-attachments/assets/743e1804-f882-4702-81d5-c4fd8e691fd3)

Reply:
Replacing `ngbDropdownToggle` with `[additionalClasses]="'dropdown-toggle'"` already resolved this

Reply:
Or like that:

![Screenshot from 2024-12-09 10-13-53](https://github.com/user-attachments/assets/6e74573d-ab76-40ca-bf6e-739c77ad54a6)